recipes/mariadb-connector-python/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "mariadb" %}
+{% set version = "1.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mariadb-{{ version }}.tar.gz
+  sha256: 3c7eebcb94588b879069a00ae3894ef404f24ed350c40d633ccb98964bc1de55
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win or osx]
+  missing_dso_whitelist:
+    - '*'
```

Comment:
You probably shouldn't use this. If you actually need it a comment should be included for why.
```suggestion
```